example/nodejs/teardown.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function () {
+
+    var path = require('path');
+    var provider = require(path.resolve(__dirname, 'provider.js'));
+
+    try {
+        provider.verify(); // Verify that all the mock calls has been made, throw error if not
```

Comment:
This is a little confusing. We're not verifying the provider here (provider verification is what happens in the provider project and is an entirely separate process). `mockProvider` would be more accurate.